templates/source/access-points.html

```diff
@@ -117,8 +117,9 @@
                 </div>
             </div>
             <div class="form-group">
-                <div class="col-sm-12">
+                <div class="col-sm-12 col-sm-offset-9">
```

Comment:
Is this offset intended to pull the buttons to the right-hand side of the page? If so, we can use the `text-right` class instead to accomplish the same behavior without customizing the grid. The offset is currently causing the `Submit` button to appear outside the bounds of the page:

<img width="945" alt="Screen Shot 2019-08-06 at 9 43 13 AM" src="https://user-images.githubusercontent.com/14170650/62549765-ac494480-b82e-11e9-8e55-ae9383932e3e.png">